api/inventorypb/agents.proto

```diff
@@ -623,6 +623,12 @@ message AddMongoDBExporterRequest {
   bool tls = 5;
   // Skip TLS certificate and hostname validation.
   bool tls_skip_verify = 6;
+  // Certificate file.
+  string tls_certificate_key_file = 10;
+  // Certificate password.
+  string tls_certificate_key_file_password = 11;
```

Comment:
TBH, not sure about that one. Storing encrypted content next to the decryption password doesn't achieve anything. It might be better to decrypt file on pmm-admin side.

Reply:
:+1:

Reply:
It will require including mongodb lib into pmm-admin and as far as I remember we are trying to keep pmm-admin as simple as possible. and it's simpler to just use `tlsCertificateKeyFile` and `tlsCertificateKeyFilePassword` in DSN

Reply:
X.509 certificate decryption can be done without mongolib for sure – all requires package are in stdlib.

I will ask around if we have use case for that

Reply:
Link to Roma response:
https://percona.slack.com/archives/G2HEW592P/p1606391990252400?thread_ts=1606376788.238000&cid=G2HEW592P